src/Components/Styles/Styles.js

```diff
@@ -1,10 +1,32 @@
 import React from 'react';
 import './Styles.css';
+import { Link } from 'react-router-dom';
 
 const Styles = () => {
   return (
     <section className="style-section-container">
+      <h2 className="about-header">Styles</h2>
       <h3 className="style-header">Traditional locs</h3>
+      <figure className="style-figure">
```

Comment:
Added photos including a figcaption to link back to the original creator on Unsplash. I will likely update the alt text in the future to be more descriptive.